src/Symfony/Component/HttpFoundation/Session/Storage/Handler/PdoSessionHandler.php

```diff
@@ -321,8 +328,6 @@ public function destroy($sessionId)
      */
     public function write($sessionId, $data)
     {
-        $maxlifetime = (int) ini_get('session.gc_maxlifetime');
```

Comment:
this removes the possibily to set a lifetime per session as I explained. So this is a BC break.